phoenix-core/src/it/java/org/apache/phoenix/end2end/AlterTableWithViewsIT.java

```diff
@@ -1216,5 +1218,248 @@ public void testDroppingIndexedColDropsViewIndex() throws Exception {
+            viewDDLTimestamp = CreateTableIT.verifyLastDDLTimestamp(schemaName, viewName,
+                viewFullName, viewDDLTimestamp, conn);
+            conn.createStatement().execute(viewColumnDropDDL);
+            //verify DDL timestamp changed because we dropped a column from the view
+            viewDDLTimestamp = CreateTableIT.verifyLastDDLTimestamp(schemaName, viewName,
+                viewFullName, viewDDLTimestamp, conn);
+            conn.createStatement().execute(tableColumnAddDDL);
+            //verify DDL timestamp DID NOT change because we added a column from the base table
+            assertEquals(viewDDLTimestamp, getLastDDLTimestamp(conn, viewFullName));
+            conn.createStatement().execute(tableColumnDropDDL);
+            assertEquals(viewDDLTimestamp, getLastDDLTimestamp(conn, viewFullName));
+        }
+    }
+
+    @Test
+    public void testLastDDLTimestampWithChildViews() throws Exception {
+        Assume.assumeTrue(isMultiTenant);
+        Properties props = new Properties();
+        String schemaName = SCHEMA1;
+        String dataTableName = "T_" + generateUniqueName();
+        String globalViewName = "V_" + generateUniqueName();
+        String tenantViewName = "V_" + generateUniqueName();
+        String dataTableFullName = SchemaUtil.getTableName(schemaName, dataTableName);
+        String globalViewFullName = SchemaUtil.getTableName(schemaName, globalViewName);
+        String tenantViewFullName = SchemaUtil.getTableName(schemaName, tenantViewName);
+
+        String tableDDL = generateDDL("CREATE TABLE IF NOT EXISTS " + dataTableFullName + " ("
+            + " %s ID char(1) NOT NULL,"
+            + " COL1 integer NOT NULL,"
+            + " COL2 bigint NOT NULL,"
+            + " CONSTRAINT NAME_PK PRIMARY KEY (%s ID, COL1, COL2)"
+            + " ) %s");
+
+        //create a table with a child global view, who then has a child tenant view
+        String globalViewDDL =
+            "CREATE VIEW " + globalViewFullName + " AS SELECT * FROM " + dataTableFullName;
+
+        String tenantViewDDL =
+            "CREATE VIEW " + tenantViewFullName + " AS SELECT * FROM " + globalViewFullName;
+
+        long startTS = EnvironmentEdgeManager.currentTimeMillis();
+        long tableDDLTimestamp, globalViewDDLTimestamp;
+
+        try (Connection conn = DriverManager.getConnection(getUrl())) {
+            conn.createStatement().execute(tableDDL);
+            conn.createStatement().execute(globalViewDDL);
+            tableDDLTimestamp = getLastDDLTimestamp(conn, dataTableFullName);
+            globalViewDDLTimestamp = getLastDDLTimestamp(conn, globalViewFullName);
+        }
+        props.setProperty(TENANT_ID_ATTRIB, TENANT1);
+        try (Connection tenantConn = DriverManager.getConnection(getUrl(), props)) {
+            tenantConn.createStatement().execute(tenantViewDDL);
+        }
+        // First, check that adding a child view didn't change the timestamps
```

Comment:
nit: could you please change the comment to be more specific.
` // First, check that adding a child view didn't change the timestamps of base table `

Reply:
@gjacoby126  looks like this comment was missed ?